WinImageBuilder.psm1

```diff
@@ -582,15 +574,6 @@ function Create-VirtualSwitch
     return $sw
 }
 
-function Check-Prerequisites
-{
-    $needsHyperV = Get-WindowsOptionalFeature -Online -FeatureName Microsoft-Hyper-V
-    if ($needsHyperV.State -ne "Enabled"){
-        Write-Warning $noHypervWarning
-        exit 1
-    }
-}
-
 function GetOrCreateSwitch
 {
     $vmSwitches = Get-VMSwitch -SwitchType external

```

Comment:
Is not present on which server versions? It's available on Hyper-V, it's available on WS2016, ...